### DISCUSSION TITLE
terra-chromeos enterperise enrollment.

### DISCUSSION THREAD

**Discussion by leoisbadatmakeingthings** [3kh0/ext-remover#1103] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is there any way to get around enterprise enrollment with terra-chromeos? I cant use it because of enterprise enrollment forcing me to update, which doesn't work. Is there any way to bypass the update screen or enterprise enrollment. I do have access to the dev console and root.

**Comment by S-PScripts** [comment]
Can you do CAUB

**Reply by leoisbadatmakeingthings** [reply]
I'm stuck on the setup I cant launch a web browser or login to a graphical interface, I cant even update it just errors if i try.

**Comment by leoisbadatmakeingthings** [comment, marked as answer]
Fixed. Had to hack together a fakemurk script to work with a non NVME drive, however doing that broke both the USB with TerraOS on it and the chromeos on the internal storage (probably because of the format command) and so I had to back up the partition, restore the TerraOS image to my USB again, restore the partition and then it booted into setup but it was super laggy and would constantly freeze. When it froze I hard rebooted it, booted back into the USB and each time I got further into the setup until I completed it and was able to get the ChromeOS partition working. You can find the modified fakemurk script at memcat.pages.dev/fakemurk.sh
